### ISSUE TITLE
timedate completion

### ISSUE THREAD

**Issue by yankeeinlondon** [blcook223/DummyData#1]
**Feature Request**

It's very common to have time and date together as one property. it would be also be REALLY nice to have the ability to format the datetime string to the standard ISO and regional formats.

> {% datetime "YYYY-MM-DD H:m:s" %}

Having this would actually allow you to remove both date and time as the format control would make the _datetime_ do both (although for convenience and backward compatibility it's probably better to keep them in the API). Great work with this, useful immediately!

**Comment by blcook223** [comment]
Ken,

Thanks for this excellent suggestion as well as issues #2 and #3. I've been out of town, but should have time to work on all three in the next two weeks.

**Comment by yankeeinlondon** [comment]
Great! Love what's there already.

**Comment by blcook223** [comment]
I've added a `datetime` function, which will be included in the next release. Cf. 96b6b97a810886d4c43101f1a3f9926bcbf31dbf.

I've also added the ability to specify a format string for the `date`, `time`, and `datetime` functions. See the [Python `datetime` documentation](https://docs.python.org/2/library/datetime.html#strftime-strptime-behavior) for details.

Format strings will have to be enclosed by escaped quotation marks, like this:

```
"{% datetime \"%m/%d/%Y\" %}"
```
